lib/london_bike_hire_cli/basic_renderer.rb

```diff
@@ -1,38 +1,31 @@
+require 'erb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.